packages/database/src/index.js

```diff
@@ -0,0 +1,61 @@
+const sqlite3 = require('sqlite3').verbose();
+
+exports.initDB =  function initDB(path) {
```

Comment:
There is some spacing oddities. You can download prettier and run `prettier --write .` at the root of this package, and it will clean them up for you.